### ISSUE TITLE
Design Attributes 

### ISSUE THREAD

**Issue by KyletheWolff** [TheLearningCurve/RevampNutritionAPI#38]
What colors and font should we be using...

**Comment by BrandonVanderMey** [comment]
Something besides the default lol

**Comment by bradleynelson** [comment]
Roboto is the default font for Android.

**Comment by bradleynelson** [comment]
![nutrition-api](https://cloud.githubusercontent.com/assets/12861929/8243294/dc5721f2-15e3-11e5-8e81-f53952312b5b.png)
Thoughts?

**Comment by KyletheWolff** [comment]
New name lol the name of the project is not the true name we should use

**Comment by bradleynelson** [comment]
What's it matter?  Apps are called retarded things all the time, this is at least related.  Besides, you never gave me a name, customer.

**Comment by bradleynelson** [comment]
![mockup](https://cloud.githubusercontent.com/assets/12861929/8243988/3bc36f5a-15ea-11e5-8865-08a41d8d6e2e.png)
You still haven't given me any requirements.

**Comment by KyletheWolff** [comment]
I am working on it lol

**Comment by KyletheWolff** [comment]
I like the nutrition icon

**Comment by KyletheWolff** [comment]
![design](https://cloud.githubusercontent.com/assets/10425034/8244096/2ac7450e-15eb-11e5-8975-f421065c6552.JPG) 

Here is the design so far with the nav menu

**Comment by bradleynelson** [comment]
But you don't even know if you need a nav menu at this point right?

**Comment by KyletheWolff** [comment]
We will. We decided that we will add some more features

**Comment by bradleynelson** [comment]
Alright, so you want a header with a hamburger icon and a search field.  Then two output fields?  The Nav Slider should come in over the app.  Material has the title of the app in the header.  Shouldn't be too hard, the output fields will have to be their own elements because everything in Material is designed that way, as if you ripped out pieces of paper and laid them on top of each other.

**Comment by bradleynelson** [comment]
Those are light green from google's color palette, do you like that color of green?  Do you want blue? teal?

**Comment by KyletheWolff** [comment]
I like the green. so far this is what we want: 

static header bar: Menu Button, and SearchField
Underneath is the nav Menu and to the right of that is the item panel(LEFT) and nutritionLabelPanel(RIGHT)

We need an image or something to put in the Label because nothing will be there until the user selects an item

**Comment by KyletheWolff** [comment]
![typeahead](https://cloud.githubusercontent.com/assets/10425034/8244488/5aefd4ea-15ef-11e5-90c6-42319890b5a7.JPG)

We need design for typeahead view

![itemlist](https://cloud.githubusercontent.com/assets/10425034/8244490/5d257ada-15ef-11e5-9dfd-6e89e9b54557.JPG)

We need a design for the item list view I like what we are doing now but we have a few things: 
Normal State - Where there is no interaction at all
Hover State - I want it to pop out in a way (Drop Shadow - implemented) 
Click State - I am open to ideas
I want an (i) symbol or something that a user can see more details : 

`{
"item_name": "Pizzaghetti* - Pepperoni Pizza",
"brand_name": "Boston Pizza",
"thumbnail": "https://d1r9wva3zcpswd.cloudfront.net/533d7cc35b8015402e5ce10d.jpg",
"nutrient_name": "Calories",
"nutrient_value": 760,
"nutrient_uom": "kcal",
"serving_qty": 411,
"serving_uom": "g",
"resource_id": "JayDhyDo6",
"nutrients": null
}`

That is what we get back from the service. We use the brand and item name and logo for the main item. I want to use almost everything else for the detail view.

**Comment by bradleynelson** [comment]
![mockup1](https://cloud.githubusercontent.com/assets/12861929/8244553/2f52e682-15f0-11e5-85b3-9671d899ed15.png)
Here's just a quick mockup of a basic material design... We could make each item a card.

**Comment by KyletheWolff** [comment]
What would the items look like? The entire card?

**Comment by bradleynelson** [comment]
no, we could make tiles within that card... we could either do rows and columns like 3 wide and like 3 down or w/e it fills (probably want it scrollable) or just do line items like coupons in the Meijer app.

**Comment by bradleynelson** [comment]
![mockup-typeahead](https://cloud.githubusercontent.com/assets/12861929/8263190/6be3eea0-16a6-11e5-96c2-370772e6c08c.png)

**Comment by KyletheWolff** [comment]
Font?

**Comment by bradleynelson** [comment]
I need to change that... Font is Roboto 87% color of #000000 which is primary color for Material Design font.

**Comment by bradleynelson** [comment]
![mockup-typeahead](https://cloud.githubusercontent.com/assets/12861929/8263391/0bdccd54-16a8-11e5-8c5c-315ed432c6d3.png)
Header #689F38  Background #F9F9F9  Fields #FFFFFF  Active #76FF03  Divider #000000-12%

**Comment by bradleynelson** [comment]
the font should look sharper in app too, for some reason it's looking like poop in PhotoShop.

**Comment by bradleynelson** [comment]
Something else you could do too is make the font larger on hover, I did not depict that, but if you feel like you need a more active state that is an option.
